evaluations/yieldscan_1_mmagician.md

```diff
@@ -0,0 +1,44 @@
+|    0d. | Article/Tutorial           | <ul><li>[x] </li></ul> | <ul><li>[YieldScan Help Center](https://intercom.help/yieldscan)</li></ul>                                                                                                                                                                                                                                                                                                           | Tutorial videos are helpful                                                                                                                                   |
+|     1. | Controller Account Support | <ul><li>[ ] </li></ul> | https://yieldscan.app/                                                                                                                                                                                                                                                                                                                                                               |                                                                                                                                                               |
+|     2. | Ledger Wallet Support      | <ul><li>[ ] </li></ul> | https://yieldscan.app/                                                                                                                                                                                                                                                                                                                                                               |                                                                                                                                                               |
+|     3. | Docker                     | <ul><li>[x] </li></ul> | <ul><li>[Docker file](https://hub.docker.com/r/sahilnanda/yieldscan-frontend)</li><li>[Docker command](https://github.com/yieldscan/yieldscan-frontend/tree/develop#docker)</li></ul>                                                                                                                                                                                                |                                                                                                                                                               |
+
+## General notes
+
+A lot of code duplication between the crawler and the api repos, probably everything under `src/models` & `src/interfaces` & large parts of other sub-directories. Would be good to consider using for example git submodules to share the code.
+
+Could you also point to the exact location of the patch used for going around ledger's batch Tx's please?
+
+### Frontend
+
+Cannot run the frontend with `yarn dev`. Getting the following errors:
```

Comment:
have pushed a fix for this

Reply:
It's still failing for me with:
```
error - SyntaxError: Unexpected token # in JSON at position 5
    at JSON.parse (<anonymous>)
    at eval (webpack-internal:///./yieldscan.config.js:81:61)
    at Array.filter (<anonymous>)
    at getAllNetworks (webpack-internal:///./yieldscan.config.js:81:38)
    at eval (webpack-internal:///./lib/store.js:54:91)
    at Module../lib/store.js (/Users/marcin/dev/yieldscan/yieldscan-frontend/.next/server/pages/index.js:55:1)
    at __webpack_require__ (/Users/marcin/dev/yieldscan/yieldscan-frontend/.next/server/webpack-runtime.js:33:43)
    at eval (webpack-internal:///./components/common/layouts/base.js:7:68)
    at Module../components/common/layouts/base.js (/Users/marcin/dev/yieldscan/yieldscan-frontend/.next/server/pages/index.js:22:1)
    at __webpack_require__ (/Users/marcin/dev/yieldscan/yieldscan-frontend/.next/server/webpack-runtime.js:33:43) {
  page: '/'
}
```

Reply:
@mmagician  apologies for the inconvenience

I believe it is because of inline comment in the `.env` file in the readme file, can you try with the below `.env`. Let me know if it works and i will push the fix for this.

```
# Main API endpoint
NEXT_PUBLIC_API_BASE_URL="http://localhost:5000/api"
# example: NEXT_PUBLIC_API_BASE_URL="http://localhost:5000/api"

# Manage Testnets
# if true westend will be enabled
NEXT_PUBLIC_TESTNETS_ENABLED=true

# Network Endpoints
NEXT_PUBLIC_POLKADOT="wss://polkadot.api.onfinality.io/public-ws"
NEXT_PUBLIC_KUSAMA="wss://kusama-rpc.polkadot.io"
NEXT_PUBLIC_WESTEND="wss://westend-rpc.polkadot.io"
```

Reply:
> Let me know if it works and i will push the fix for this.

Updated readme for both [main](https://github.com/yieldscan/yieldscan-frontend/tree/master) and [dev](https://github.com/yieldscan/yieldscan-frontend/tree/develop) branches.

Reply:
yeah that worked, thanks